src/content/reference/gsctl/show-cluster.md

```diff
@@ -39,6 +39,8 @@ Worker nodes running:          8
 Worker EC2 instance type:      m5.xlarge
 CPU cores in workers:          28
 RAM in worker nodes (GB):      26.5
+Labels:                        usage=staging
+                               locked=false
```

Comment:
I'd place the `Labels` row(s) under `Release version`. That way the size and resource group keeps at the end.

By the way, that example is for release version prior to v10.x.x (non node pools). For >10 it looks slightly different, as there is less worker node information.